examples/aws_iot_native_networking.py

```diff
@@ -118,6 +116,6 @@ def message(client, topic, msg):
 # NOTE: NO code below this loop will execute
 # NOTE: Network reconnection is NOT handled within this loop
 while True:
-    aws_iot.loop()
+    aws_iot.loop(10)
```

Comment:
@jersu11  What's the reason for waiting 10 (ms? seconds?) here?

Reply:
@brentru , in the example code, it's to prevent an error

The example code in this directory inits the MQTT client with just a few params. If the param 'socket_timeout' is not set, the mqtt client will default that value to 1

https://github.com/adafruit/Adafruit_CircuitPython_MiniMQTT/blob/f871143d3f160ce9978df9b58565eeb9fb60894e/adafruit_minimqtt/adafruit_minimqtt.py#L156

So, with an mqtt client default 'socket_timeout' of 1, when the aws_iot calls the mqtt client `loop()` method without a 'timeout' value, it will default to the value 0

https://github.com/adafruit/Adafruit_CircuitPython_MiniMQTT/blob/f871143d3f160ce9978df9b58565eeb9fb60894e/adafruit_minimqtt/adafruit_minimqtt.py#L943

Within the loop function, `if timeout < self._socket_timeout:` will raise an MMQTTException. So, the 10 second value in the example code is to prevent that error from raising.

Reply:
Thanks for the explanation!